frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualPaymentResponse.tsx

```diff
@@ -0,0 +1,71 @@
+import { Stack, Text } from '@chakra-ui/react'
+
+import { useIndividualPaymentSubmission } from './queries'
+
+export const IndividualPaymentResponse = (submissionId: {
+  submissionId: string
+}): JSX.Element => {
+  if (!submissionId) throw new Error('Missing submissionId')
+
+  // TODO: get paymentData from IndividualResponsePage through submission endpoint
```

Comment:
sorry I don't understand / have forgotten what this TODO is about! can provide more context?

Reply:
Ah, cause currently we are getting payment data from `submissions/:submissionId/payment` endpoint. 

However, ideally, the payment data should have been retrieved together with all the other data from `submissions/:submissionId`. So eventually we will be passing the entire paymentdata as a parameter to IndividualPaymentResponse.

Reply:
ah gotcha, so after the future PR

Reply:
yup! This issue #5864